src/main/java/teammates/ui/controller/StudentCommentsPageData.java

```diff
@@ -51,4 +113,139 @@ public String getRecipientNames(Set<String> recipients){
+                            fsName, courseId, createFeedbackQuestionTables(feedbackResultBundles.get(fsName),
+                                                                           roster));
+            
+            feedbackSessionRows.add(feedbackSessionRow);
+        }
+    }
+    
+    private List<QuestionTable> createFeedbackQuestionTables(
+            FeedbackSessionResultsBundle feedbackResultBundle, CourseRoster roster) {
+        List<QuestionTable> feedbackQuestionTables = new ArrayList<QuestionTable>();
+        
+        for (Map.Entry<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> responseEntries 
+                     : feedbackResultBundle.getQuestionResponseMap().entrySet()) {
+            
+            FeedbackQuestionAttributes feedbackQuestion = responseEntries.getKey();
+            int questionNumber = feedbackQuestion.questionNumber;
+            String questionText = feedbackResultBundle.getQuestionText(feedbackQuestion.getId());
+            Map<String, FeedbackQuestionAttributes> questions = feedbackResultBundle.questions;
+            
+            FeedbackQuestionAttributes question = questions.get(feedbackQuestion.getId());
+            FeedbackQuestionDetails questionDetails = question.getQuestionDetails();
+            String additionalInfo = questionDetails.getQuestionAdditionalInfoHtml(question.questionNumber, "");
+            
+            QuestionTable feedbackQuestionTable = 
+                    new QuestionTable(questionNumber, questionText, additionalInfo, 
+                                      createFeedbackResponseRows(feedbackResultBundle, questionDetails, 
+                                                                 responseEntries, roster));
+            feedbackQuestionTables.add(feedbackQuestionTable);
+        }
+        return feedbackQuestionTables;
+    }
+    
+    private List<ResponseRow> createFeedbackResponseRows(
+            FeedbackSessionResultsBundle feedbackResultBundle, FeedbackQuestionDetails questionDetails, 
+            Map.Entry<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> responseEntries,
+            CourseRoster roster) {
+        List<ResponseRow> feedbackResponseRows = new ArrayList<ResponseRow>();
+        
+        for (FeedbackResponseAttributes responseEntry : responseEntries.getValue()) {
+            String giverName = feedbackResultBundle.getGiverNameForResponse(
+                                                            responseEntries.getKey(), responseEntry);
+            String giverTeamName = feedbackResultBundle.getTeamNameForEmail(responseEntry.giverEmail);
+            giverName = feedbackResultBundle.appendTeamNameToName(giverName, giverTeamName);
+
+            String recipientName = 
+                    feedbackResultBundle.getRecipientNameForResponse(responseEntries.getKey(), responseEntry);
+            String recipientTeamName = feedbackResultBundle.getTeamNameForEmail(responseEntry.recipientEmail);
```

Comment:
Should be best standardising the form of line breaking for this and `giver*` above.
Break at either the `(` or after the `=`.